### PR TITLE
fix usage of varargs in ScKeynodes

### PR DESCRIPTION
**Pull request by rudi-kit**

### CODE REVIEW

**Review by NikitaZotov — APPROVED** [review]

File: tests/api_test.py [hunk 1]
```diff
@@ -323,6 +327,27 @@ def test_template_generate(self):
...
 
 
+class TestScKeynodes(ScTest):
+
+    class TestEnum(Enum):
+        IDTF_1 = 'idtf_1'
+        IDTF_2 = 'idtf_2'
+
+
+    def test_should_get_keynode(self):
```
Comment (AlexandrZagorskiy) [inline tests/api_test.py:337]: If I will use an incorrect identifier, what should ScKeynodes class return? Can you add one more test for such situation?

**Review by MikhailSadovsky — APPROVED** [review]

File: tests/api_test.py [hunk 2]
```diff
@@ -323,6 +327,27 @@ def test_template_generate(self):
...
+        self.get_server_message('{"id": 1, "event": false, "status": true, "payload": [1183238, 2]}')
+        keynodes = ScKeynodes()
+        keynodes.resolve_identifiers([self.TestEnum])
+        assert keynodes[self.TestEnum.IDTF_1.value].value == 1183238
+        assert keynodes[self.TestEnum.IDTF_2.value].value == 2
+
+
 client_test_cases = (
     TestClientCreateElements,
     TestClientCheckElements,
```
Comment (AlexandrZagorskiy) [inline tests/api_test.py:359]: CI was failed, please fix errors

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: tests/api_test.py (hunk 1 above)
Comment (rudi-kit) [reply tests/api_test.py:337]: done

File: tests/api_test.py (hunk 2 above)
Comment (rudi-kit) [reply tests/api_test.py:359]: fixed

File: README.md [hunk 3]
```diff
@@ -112,12 +112,12 @@ status # True
 Get the content of the given link.
 
 *Parameters*: An ScAddr class object.
```
Comment (AlexandrZagorskiy) [inline README.md:114]:
```suggestion
*Parameters*: ScAddr class objects.
```

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**